code/socialDistribution/models.py

```diff
@@ -106,33 +102,40 @@ class PostContentType(models.TextChoices):
         PNG = 'PNG', 'image/png;base64'
         JPEG = 'JPEG', 'image/jpeg;base64'
 
-    class PostVisibility(models.TextChoices):
-        PUBLIC = "PB", "PUBLIC"
-        FRIENDS = "FRD", "FRIENDS"
+    TITLE_MAXLEN = 50
+    DESCRIPTION_MAXLEN = 50
+    CONTEXT_TEXT_MAXLEN = 200
+    CONTENT_MEDIA_MAXLEN = 1000
 
-    title = models.CharField(max_length=50)
+    title = models.CharField(max_length=TITLE_MAXLEN)
     source = models.URLField(max_length=200)
```

Comment:
Minor thing. I think you intended to use  CONTEXT_TEXT_MAXLEN that you introduced above instead of 200.

Reply:
Oh nice catch! Thanks 👌
I'll create URL_MAXLEN, which is 2048 (the recommended max len for urls)